Sources/CodeGen/CXX/CXXCodeWriter.swift

```diff
@@ -4,7 +4,12 @@ import ValModule
 public struct CXXCodeWriter {
 
   /// Initializes the current object.
-  public init() {}
+  public init(formatter: @escaping CodeTransform = identity()) {
```

Comment:
```suggestion
  public init(formatter: CodeTransform? = nil) {
```

IIUC, that's what your `identity` function is representing: the absence of any transformation.